package.json

```diff
@@ -6,8 +6,20 @@
         "build": "vite build"
     },
     "devDependencies": {
+        "@tailwindcss/forms": "^0.5.2",
+        "alpinejs": "^3.4.2",
+        "autoprefixer": "^10.4.2",
         "axios": "^1.1.2",
         "laravel-vite-plugin": "^0.8.0",
+        "postcss": "^8.4.6",
+        "postcss-scss": "^4.0.7",
+        "tailwindcss": "^3.1.0",
         "vite": "^4.0.0"
+    },
+    "dependencies": {
+        "bootstrap": "^5.3.1",
+        "jquery": "^3.7.0",
+        "material-icons": "^1.13.10",
+        "postcss-loader": "^7.3.3"
```

Comment:
I don't think these are added by breeze, at least from what I  tested. Maybe by livewire? But I think we should do this step by step.